kafka-streams-partitioners/weighted-group-partitioner/src/main/java/org/hypertrace/core/kafkastreams/framework/partitioner/KeyHashPartitioner.java

```diff
@@ -0,0 +1,18 @@
+package org.hypertrace.core.kafkastreams.framework.partitioner;
+
+import java.util.concurrent.atomic.AtomicInteger;
+import org.apache.kafka.common.utils.Utils;
+import org.apache.kafka.streams.processor.StreamPartitioner;
+
+public class KeyHashPartitioner<K, V> implements StreamPartitioner<K, V> {
+  private final AtomicInteger currentPartition = new AtomicInteger();
```

Comment:
where is this used?

Reply:
Fixed. Not required. Started off with a little different implementation in mind.